cockatrice/src/tab_game.cpp

```diff
@@ -837,6 +837,8 @@ void TabGame::sendGameCommand(PendingCommand *pend, int playerId)
     AbstractClient *client = getClientForPlayer(playerId);
     if (!client)
         return;
+
+    connect(pend, SIGNAL(finished(Response, CommandContainer, QVariant)), this, SLOT(commandFinished(const Response &)));
```

Comment:
Aren't you connecting this signal twice? Once here and once in the method calling this one? Can you explain?